hack/jenkins/e2e.sh

```diff
@@ -116,40 +117,24 @@ GCE_SLOW_TESTS=(
 
 # Tests which are not able to be run in parallel.
 GCE_PARALLEL_SKIP_TESTS=(
-    "Etcd"
-    "NetworkingNew"
-    "Nodes\sNetwork"
-    "Nodes\sResize"
-    "MaxPods"
-    "ResourceUsage"
-    "SchedulerPredicates"
-    "Services.*restarting"
-    "Shell.*services"
+    "Etcd"                  # simulate Etcd failures.
+    "Nodes\sNetwork"        # simulate node network failures.
+    "Nodes\sResize"         # add and remove nodes to and from the cluster
+    "MaxPods"               # saturates nodes with pods
+    "ResourceUsage"         # TODO: Not clear why this is here?
+    "SchedulerPredicates"   # TODO: Perhaps enable some of these?
+    "restarting\skube-proxy"# breaks pod and service networking
     )
 
 # Tests which are known to be flaky when run in parallel.
 GCE_PARALLEL_FLAKY_TESTS=(
-    "DaemonRestart"
-    "Elasticsearch"
-    "PD"
-    "ServiceAccounts"
-    "Services.*change\sthe\stype"
-    "Services.*functioning\sexternal\sload\sbalancer"
-    "Services.*identically\snamed"
     "Services.*release.*load\sbalancer"
-    "Services.*endpoint"
```

Comment:
It's possible that it's been broken in the past few hours, after this PR was submitted, but it had a very clean bill of health before that:
job/kubernetes-e2e-gce-parallel/7717/testReport/(root)/Kubernetes%20e2e%20suite/Service_endpoints_latency_should_not_be_very_high/history/

But besides that, the regex that is being removed never matched that test anyway :-)